src/makeInfinityAdder.js

```diff
@@ -7,6 +7,23 @@
 
 function makeInfinityAdder() {
   // write code here
+  let sum = 0;
+
+  const adder = (arg) => {
```

Comment:
It is better to change `arg` to something like `num`/`number`/`n`, something similar to number or math)